deploy/helm/sumologic/values.yaml

```diff
@@ -85,6 +85,10 @@ sumologic:
   # If enabled, collect K8s events
   eventCollectionEnabled: true
 
+  events:
```

Comment:
should we move `eventCollectionEnabled` under this, maybe as:
```
events:
  collectionEnabled: true
  sourceCategory: ""
```
?

Reply:
this will be changed after #349

Reply:
I'm confused, #349 is already merged

Reply:
oh sorry wrong one, it is #330, specifically [this change](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/330/files#diff-1afd7b203a6373f36b3fdb4d68d7ddb5R273)

Reply:
Great, thanks for confirming